packages/pwa-buildpack/src/WebpackTools/PWADevServer.js

```diff
@@ -85,6 +87,9 @@ const PWADevServer = {
                         })
                     )
                 );
+            },
+            before(app) {
+                addImgOptMiddleware('/scale', app, config.env);
```

Comment:
The `/scale` value here feels a bit fragile. It looks like it's coupled to the `ImageOpt` configuration in `/packages/venia-concept/venia-upward.yml` but I had to go up and down the diff a few times to work out what was connected where. Could this be specified in that configuration instead?

Reply:
That's a good idea. We should push more config to `venia-upward.yml` in general.

We have two goals here that are sometimes opposed:

- To centralize config on a single resolution path and contract (env vars -> UPWARD  YAML -> defaults -> code)
- To be as zero-config as possible

It makes sense to me to have a sensible default route for the onboard, dev-and-staging-only image opto service. But it should definitely be clearly documented, and there should be a very friendly configuration debugging mode, where you can see what comes from where.